packages/accounts/plugindefs/multisig/config.ts

```diff
@@ -0,0 +1,16 @@
+import { sepolia } from "@alchemy/aa-core";
+import { parseAbiParameters } from "viem";
+import type { PluginGenConfig } from "../types";
+import { MultisigPluginAbi } from "./abi.js";
+
+export const MultisigPluginGenConfig: PluginGenConfig = {
+  name: "MultisigPlugin",
+  abi: MultisigPluginAbi,
+  addresses: {
+    [sepolia.id]: "0x0b135A12EB2f7b441DCcE5F7DE07DB65AE7c4649",
```

Comment:
we're not deployed on other chains yet right?

Reply:
correct, we will only do that after audit finalization